src/lib/y2partitioner/dialogs/partition_table_type.rb

```diff
@@ -47,7 +52,7 @@ def label
         # @macro seeAbstractWidget
         def help
           # helptext
-          _("<p>Choose the partition table type for the new partition.</p>")
+          _("<p>Choose the type for the new partition table.</p>")
```

Comment:
is it "type for the new" or "type of the new"?

Reply:
Both sounds equally valid to me. I just respected the previous one, but I can change it to "of" if you prefer so. :-)

Reply:
@mvidner your fine tuned language sense here would be great :wink: